src/System.Management/src/System/Management/ManagementClass.cs

```diff
@@ -0,0 +1,1475 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Specialized;
+using System.Runtime.InteropServices;
+using System.Runtime.Serialization;
+using System.CodeDom;
+
+namespace System.Management
+{
+	//CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC//	
```

Comment:
This file is using tabs instead of spaces. Presumably the running the code formatter will fix?